src/util/buildTransactions.ts

```diff
@@ -174,7 +174,7 @@ export class BuildTransactions {
 
     aggregateComplete(innerTxn: InnerTransaction[]): AggregateTransaction {
 
-        return this.transactionBuilderFactory.aggregateCompleteV1()
+        return this.transactionBuilderFactory.aggregateCompleteV1 ()
```

Comment:
no extra spacing before function invoke,
if it is eslint, please set it to off in .eslintrc file
```
"rules": {
  "func-call-spacing": "off"
}
```

Reply:
resolved